packages/react/src/primitives/Breadcrumbs/BreadcrumbsContext.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+interface BreadcrumbsContextInterface {
```

Comment:
Prefer appending `Type` over `Interface` for Context types namespaces (existing convention)